src/connector/SpectodaNodeSerialConnector.ts

```
@@ -0,0 +1,1135 @@
+            await this.#write(CHANNEL_DEVICE, bytes, 10000);
+            written += index_to - index_from;
+
+            const percentage = Math.floor((written * 10000) / firmware_bytes.length) / 100;
+            logging.info(percentage + "%");
+
+            this.#runtimeReference.emit("ota_progress", percentage);
+
+            index_from += chunk_size;
+            index_to = index_from + chunk_size;
+          }
+        }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)